man/repart.d.xml

```diff
@@ -648,6 +648,27 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[Verity] Section Options</title>
```

Comment:
i am not entirely sure this should a new section. we have only very vague rules about when a new [] section should be added or when not. but i think just calling these fields VerityDataBlockSizeBytes= and sticking it in the normal [Partition] section would be more alike what we so far did.

Reply:
So I figured we might want add more verity options later and maybe start exposing individual filesystem options as well. Do we really want to jam all of those into the Partition section?

Reply:
Can someone let me know what the definitive answer is on this one and I'll update. I've got either:

1. A `[Verity]` section with options `DataBlockSizeBytes=` and `HashBlockSizeBytes=`
2. No new sections and options `VerityDataBlockSizeBytes=` and `VerityHashBlockSizeBytes=`

Reply:
Let's keep it in `Partition` section for now